api/types/types.go

```diff
@@ -600,3 +600,10 @@ type BuildCache struct {
 	Parent      string
 	Description string
 }
+
+// BuildCachePruneOptions hold parameters to prune the build cache
+type BuildCachePruneOptions struct {
+	All          bool
+	KeepDuration time.Duration
+	KeepStorage  float64 // in MB
```

Comment:
@AkihiroSuda @vdemeester I'm not sure if I want `KeepStorage` here or `KeepBytes` like in Buildkit API, though on the CLIs it's `--keep-storage`. Let me know what you think.

Reply:
either is fine, but I think the type should be `int64` in bytes

Reply:
@tiborvass same as @AkihiroSuda :stuck_out_tongue_closed_eyes:, `KeepStorage` is fine :wink: